docs/books/admin_guide/09-backups.md

```diff
@@ -463,13 +463,13 @@ tar xvfP /backups/etc.133.P.tar
 
 !!! Warning
 
-    Go to the right place.
+    For security reasons, caution should be exercised when extracting backup files saved in absolute mode.
 
-    Check the contents of the backup.
+    Once again, it should be emphasized that before performing extraction operations, you should always check the contents of the backup files (especially those saved in absolute mode)
 
```

Comment:
Again, passive voice is added with your change. You could use:

"Once again, before performing extraction operations, you should always check the contents of the backup files (particularly those saved in absolute mode)."